Stop using ruby version from outside docker

saraycp opened the pull request:
Since we moved from Tumbleweed to Leap, the ruby version inside the container is 2.5, however, the binstubs were created using the ruby version from the machine (outside Docker) which sometimes was different.

To avoid problems with different setups in the operators' machines, we make ansible-obs self-contained.

~We also take the opportunity to simplify the paths, replacing `/home/${USER}/ansible-obs` by `/home/ansible-obs`.~

Co-authored-by: Eduardo Navarro <enavarro@suse.com>

dmarcoux commented on Dockerfile:
```diff
@@ -1,15 +1,23 @@
 
 RUN useradd -l -u ${USER_ID} -g users --home-dir /home/${USER_NAME} --create-home ${USER_NAME}
 
+RUN gem install bundler
```
To ensure reproducible Docker images, we should pin the version of `bundler` we install.

dmarcoux commented:
The _README.md_ needs to be updated to remove `--build-arg USER_ID=$(id -u)` from the `docker-compose` command example.

saraycp commented:
> The _README.md_ needs to be updated to remove `--build-arg USER_ID=$(id -u)` from the `docker-compose` command example.

@dmarcoux I think you meant `--build-arg GROUP_ID=$(id -u)` here, which was the argument we removed from the code. I updated the README.md accordingly.

We updated the code including these changes:
- We finally decided not to change the paths, it ended up being more confusing not having a `/home/<username>`
- We pin the version of bundler

Please have another look

saraycp commented:
BTW, the last deployment was done using this branch. So this works!
